docs/Troubleshooting.md

```diff
@@ -25,13 +25,85 @@ Brim invokes to extract flows from your pcap handles the most common variations
 we've encountered, but you may have come across a corner case that it wasn't
 prepared to handle.
 
-If the Brim app shows an error message, please [open a bug report](#opening-an-issue)
-and include a [screenshot](#screenshots-videos) that includes the error message
-or cut & paste the error message text into the issue.
-
-If you don't see an error message, or the error seems terse and unhelpful, it
-may help us get to the bottom of it faster if you include the debug info from
-[Developer Tools](#developer-tools).
+Such problems are often due to file format variations that are unique to the
+system that generated the capture. In such situations, the most expedient debug
+path would be to send us the capture file so our developers can use it to
+reproduce the problem, fix it, and create automated tests to ensure the problem
+doesn't arise again in the future. We understand that concerns regarding
+[sensitive information](#sensitive-information-important)
+may make it difficult or impossible to share such data. If the specific
+capture with which you first saw the problem cannot be shared, an alternative 
+approach would be to share a special debug capture generated by the same
+system, but made up of traffic known to be innocuous (e.g. performing a
+`curl` to a common public web site). If you're able to share your original
+capture file, or can share a special debug capture that also reproduces the
+problem, please [open a bug report](#opening-an-issue) and attach it along with
+a [screenshot](#screenshotsvideos) or cut & paste of any error message you saw
+in Brim. These alone should be adequate for us to start working on a fix.
+However, if you cannot share your data, proceed with the following steps.
+
+First, if you don't see an error message in Brim, or the error seems terse and
+unhelpful, it may help us get to the bottom of it faster if you include the
+debug info from [Developer Tools](#developer-tools).
+
+Beyond that, the most comprehensive way to debug is to use the
+[zq](https://github.com/brimsec/zq) toolset at the command line to perform
+steps that are functionally equivalent to those initiated by the Brim
+application to extract flows from your pcap. This will allow you to better
```

Comment:
Just a suggestion: say up front that if you go through this process, you expect it to fail. You mention this after all the steps, but I think its worth mentioning in some form beforehand, like:
```suggestion
application to extract flows from your pcap, and report at which point you see a failure. This will allow you to better
```

Reply:
Thanks @alfred-landrum. I adjusted the wording using your suggestion as a starting point.